app/src/androidTest/java/net/skyscanner/backpack/calendar2/BpkCalendarTest.kt

```diff
@@ -158,7 +158,7 @@ class BpkCalendarTest : BpkSnapshotTest() {
 
     Espresso
       .onView(withId(R.id.bpk_calendar_recycler_view))
-      .perform(RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(0, ViewActions.click()))
+      .perform(RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(18, ViewActions.click()))
```

Comment:
can we have some constants for these to make it easier to understand what's happening here?

Reply:
These constants are specific for each tests case so I wouldn't extract it as constants. Besides it's clear to understand its meaning from the invocation place itself - e.g. cell position here.

Reply:
Yeah I understand it's positions, but it was unclear why a random 18 position - I assume this had to do with the range changes? Maybe we could have a `RANGE_START` constant and calculate other changes from that?

Reply:
It may be easier to understand its meaning from the screenshot itself to see which dates are chosen exactly rather then look into the calculation code (which may also contain errors, in theory).

What do you think about extracting these constants into local variables with meaningful name, e.g. cellIndexOf19Jan?

Reply:
yeah i like that!